src/components/Sidebar/index.js

```diff
@@ -0,0 +1,90 @@
+import HomeOutlinedIcon from '@mui/icons-material/HomeOutlined';
+import LabelOutlinedIcon from '@mui/icons-material/LabelOutlined';
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined';
+import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
+import AccountCircleOutlinedIcon from '@mui/icons-material/AccountCircleOutlined';
+import LogoutIcon from '@mui/icons-material/Logout';
```

Comment:
You can use named imports instead of default imports

Reply:
Yes, I have used this in the new branch I created for the next feat.